### PR TITLE
Create license.md

### PR DESCRIPTION
**Pull request by alexresiga**
based on [this link](https://creativecommons.org/choose/results-one?license_code=by-nc-sa&amp;jurisdiction=&amp;version=4.0&amp;lang=en) for solving #4

### CODE REVIEW

**Review by ShanikaEdiriweera — COMMENTED** [review]
Is this all?

Is this a standard licence file?

**Comment by alexresiga** [comment]
yes, I copied the text from the link i shared above from their official site

**Comment by ShanikaEdiriweera** [comment]
@alexresiga can we add the full licence in LICENCE.md and add your PR kind of line in the README.md?
Refer https://github.com/xorann/BigWigs

**Comment by alexresiga** [comment]
now it's done

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
+Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International Public License
```
Comment (ShanikaEdiriweera) [inline LICENSE.md]: Can we add a heading for the licence ?

Comment (alexresiga) [reply LICENSE.md]: done

**Comment by alexresiga** [comment]
take a look now

**Pull request merged**